src/db/query_methods.cr

```diff
@@ -125,21 +140,38 @@ module DB
     end
 
     # Executes a *query* that expects a single row and returns it
-    # as a tuple of the given *types*.
+    # as a named tuple of the given *types*.
```

Comment:
Looks like you deleted the wrong one: the commit diff has doc changes now.

Reply:
I try one last time, then I should get some sleep :)